webapp/src/main/java/com/google/sps/servlets/UploadServlet.java

```diff
@@ -76,13 +84,56 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             // Save the file on disk
             item.write(storeFile);
 
-            // TO DO: Call validator here 
+            // TO DO: get feed name and thread specification from form
```

Comment:
I don't think the client should determine the number of threads we use to execute the validator (as this is implementation detail of the service).
I would suggest using a named constant of 1.

Reply:
Ok sure